docs/usage/shoot-networking-filter.md

```diff
@@ -31,3 +31,44 @@ spec:
 ...
 ```
 
+### Ingress Filtering
+
+By default, the networking filter only filters egress traffic. However, if you enable blackholing, incoming traffic will also be blocked.
+You can enable blackholing on a per-shoot basis.
+
+```yaml
+apiVersion: core.gardener.cloud/v1beta1
+kind: Shoot
+...
+spec:
+  extensions:
+    - type: shoot-networking-filter
+      egressFilter:
+        blackholingEnabled: true
+...
+```
+Please note that if you disable `blackholing` in an existing shoot, the associated blackhole routes will not be removed automatically. 
+To remove these routes, you can either replace the affected nodes or delete the routes manually.
+
```

Comment:
Could you please also document the boundary conditions for blackhole routing to include ingress blocking, i.e. only if the load balancer forwards traffic without changing the IPs it will work? It will also be helpful to note known load balancer implementations that do not allow this, e.g. AWS Classic load balancers and OpenStack.